src/main/java/mate/academy/model/Book.java

```diff
@@ -10,7 +10,7 @@
 
 @Entity
 @Data

```

Comment:
`@Data` annotation should generate getters/setters for you. EIther remove this annotation or remove getters and setters.

Reply:
ok, done